### PR TITLE
Wrong box anomaly trending page 

### LINKED ISSUE #2: Make wrong-box anomaly trending page and job

### LINKED ISSUE #4: Trending page for wrong-box anomaly

### PR DESCRIPTION
**Pull request by jeanconn**
## Description

Make a wrong-box acquisition anomaly trending page

Fixes #2 
Closes https://github.com/sot/ska-projects/issues/77
Closes #4 

### Unit tests

- [x] No unit tests

Independent check of unit tests by [REVIEWER NAME]
- [ ] [PLATFORM]:

### Functional tests

output from this PR is in https://cxc.cfa.harvard.edu/mta/ASPECT/wrong_box_anom/

### CODE REVIEW

File: ska_trend/wrong_box_acq/task_schedule.cfg [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+heartbeat_timeout 30000             # Maximum age of heartbeat file (seconds)
+iterations        1                 # Run once then shut down task_schedule
+print_error       1                 # Print full log of errors
+disable_alerts    0                 # Don't disable alerts since this jobs runs just once/day
+loud              0                 # Run loudly or quietly (production mode)
+
+# Data files and directories.  The *_dir vars can have $ENV{} vars which
+# get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
+
+data_dir     $ENV{SKA_DATA}/wrong_box_trend      # Data file directory
```
Comment (jeanconn) [inline ska_trend/wrong_box_acq/task_schedule.cfg:14]: I have some ambiguity here, with 'wrong_box_trend' used in the task schedule, 'wrong_box_acq' used for the module, and 'wrong_box_anom' used for the draft page I showed at SSAWG.  So just let me know what is preferred and I'll make it all match.

Comment (jeanconn) [reply ska_trend/wrong_box_acq/task_schedule.cfg:14]: Per our conversation today, I'll just update everywhere to be wrong_box_anom.

File: ska_trend/wrong_box_anom/wrong_box_anom.py [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+        "guide_tstart",
+        "mag_obs",
+        "actual_slot",
+        "actual_slot_idd",
+        "actual_slot_mag_obs",
+        "another_acq_star",
+        "agasc_star",
+    ]
+
+    manual_rows = [
```
Comment (taldcroft) [inline ska_trend/wrong_box_anom/wrong_box_anom.py:336]: Let's move away from putting stuff like this in Python code in the package. For this particular case we could make a supplementary manual data file that lives in `$SKA/www/ASPECT/wrong_box_anom` which we edit as needed. Given how these data are used, that level of process control is adequate.

The action here is capture this thought as a code comment here. The NEXT time you come to the code to add a new manual event, instead change the code to store manual events in an ECSV file. You can easily create it using this code.

**Review by taldcroft — APPROVED** [review]
Approved with a lien to add one comment for a future modification.

File: ska_trend/wrong_box_anom/wrong_box_anom.py (hunk 2 above)
Comment (jeanconn) [reply ska_trend/wrong_box_anom/wrong_box_anom.py:336]: Right, that's why I brought this up.  Though I note that I took more than a few minutes to find the manual data for kalman_watch3 even with the README file that somebody left around as breadcrumbs -- so I'm not sure if package data (if not Python code) for non-fsds packages might end up being easier to manage.

**Pull request merged**